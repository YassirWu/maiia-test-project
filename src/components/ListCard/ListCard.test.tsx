import React from 'react';
import { shallow } from 'enzyme';
import Col from 'react-bootstrap/Col';

import ListCard from './ListCard';

describe('<ListCard />', () => {
  it('should render ListCard component', () => {
    const wrapper = shallow(
      <ListCard>
        <p>first child</p>
        <p>second child</p>
        <p>third child</p>
      </ListCard>
    );
    
    expect(wrapper).toMatchSnapshot();
  });
  
  it('should renders three children inside Col bootstrap component', () => {
    const wrapper = shallow(
      <ListCard>
        <p>first child</p>
        <p>second child</p>
        <p>third child</p>
      </ListCard>
    );
    
    expect(wrapper.find(Col)).toHaveLength(3);
  });
})


