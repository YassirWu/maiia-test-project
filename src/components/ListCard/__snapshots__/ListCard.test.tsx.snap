// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListCard /> should render ListCard component 1`] = `ShallowWrapper {}`;
